Use correct GitHub casing (#6701)
umbraco/UmbracoDocs: 15/umbraco-cms/tutorials/creating-and-distributing-a-package.md

The next step is to make it a bit simpler to deploy updates to the package. It is perfectly fine to log in here and upload a new version each time. The next steps will show an easier way though.

## Pushing your package to GitHub

If you are creating a package in order to share it with others it is a great idea to also share the source code. It is the open source way.

To share it, and make it easier to manage and deploy updates we will set up a GitHub repository for the package. This tutorial assumes you know what GitHub is, and that you have an account.

> Create a fresh repo, with no readme, gitignore or license - do not choose a repository template (set to 'No Template'). On the second screen it will give you a command to push an existing repository to the new GitHub repo, should look like this but with your own user in the link:

Reviewer: ```suggestion
Create a fresh repo, with no readme, gitignore or license - do not choose a repository template (set to 'No Template'). The second screen will provide a command to push an existing repository to your new GitHub repo. It will look like this but with your own username in the link:
```

So at this point we can work on our package locally, build a new version within seconds by running the pack command and then deploy it to Our using the push command.

> Not easy enough for you? Let's try automating this entire thing with GitHub Actions then.

Reviewer: ```suggestion
Not enough? Let's try automating this entire thing with GitHub Actions then.
```

## Deploy your package using GitHub Actions

> If you think back to the beginning when we set up our sites using the Package Templates you may remember that by default you get a GitHub action installed as well.

Reviewer: ```suggestion
When we set up our sites with Package Templates, a GitHub action is installed by default.
```

If you check out the `~/.github/workflows` folder in your solution, you will see there is a readme file and a build.yml file.

> The build.yml file is used by GitHub Actions, which will perform some tasks for you when certain criteria are met. If you haven't worked with continuous integration and deployment (CI/CD) before, then this may seem like magic - but don't worry we will run through the commands.

Reviewer: ```suggestion
The build.yml file is used by GitHub Actions, which will perform some tasks for you when certain criteria are met. If you’re new to continuous integration and deployment (CI/CD), we’ll go through the commands together.
```

It sets the version of the package to be what we've set in the release tag based on a previous step.
{% endhint %}

> Below this there is another step to push the package to Our, which again is like our approach locally - except now we add the API key as a GitHub secret so it's not public to everyone.

Reviewer: ```suggestion
To push the package to Our, we add the API key as a GitHub secret to keep it private.
```

With these 2 commands and a few previous ones setting up the prerequisite build and nuget tools it is now ready to be fully automated.

> Ensure you have set a GitHub secret with the name `UMBRACO_DEPLOY_KEY` and the value of the key from Our, and then go to your local solution and uncomment the UmbPack push command in the \~/.github/workflows/build.yml file.

Reviewer: ```suggestion
Ensure you have set a GitHub secret with the name `UMBRACO_DEPLOY_KEY` and the value of the key from Our. Go to your local solution and uncomment the UmbPack push command in the \~/.github/workflows/build.yml file.
```
